packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -152,7 +152,7 @@ void main() {
     expect(getScrollOffset(tester), heldPosition);
     await gesture.up();
     // Once the hold is let go, it should still snap back to origin.
-    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 2);
+    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 3);
```

Comment:
This is intended change, right?

Reply:
yes, now it takes one more frame to update the semantics

Reply:
You just need updates the semantics at the first initial metrics?

Reply:
no and subsequence changes as well, we just lucky that scroll metrics change usually come with scroll position changes so it wasn't so obvious before.

without the this pr, the semantics for the first frame is always wrong, but we are lucky that usually gets update due to some semantics changes from the modal route.

Reply:
Got it.thanks for your explanation.